<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hydraulicLoopDialog</class>
 <widget class="QDialog" name="hydraulicLoopDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hydraulic loop properties</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelButtonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="generalGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="generalGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="loopNameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="loopName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fluidLabel">
       <property name="text">
        <string>Fluid</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="fluidComboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="pipesGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>421</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Connections</string>
   </property>
   <widget class="QTableView" name="connectionsTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>hydraulicLoopDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>hydraulicLoopDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
